ci/docker/java-jni-manylinux-201x.dockerfile

```diff
@@ -34,7 +34,7 @@ RUN vcpkg install \
 
 # Install Java
 ARG java=1.8.0
-ARG maven=3.9.3
+ARG maven=3.9.4
 RUN yum install -y java-$java-openjdk-devel && \
```

Comment:
I was thinking of splitting this single line into multiple lines to make errors explicit.